build.xml

```diff
@@ -1915,7 +1915,7 @@
             </packageset>
             <arg value="-Xdoclint:all"/><!-- remove -missing to get warnings about missing javadoc tags -->
             <arg value="-Xmaxwarns"/>
-            <arg value="700"/><!-- change from default 100 warnings; keep synced with pom.xml, other javadoc targets -->
+            <arg value="100"/><!-- change from default 100 warnings; keep synced with pom.xml, other javadoc targets -->
```

Comment:
As the comment notes, can you set the maximum number of warnings on all javadoc targets in build.xml?